src/widgets/topicFrequencyGraph.js

```diff
@@ -13,23 +15,39 @@ export default async function topicFrequencyGraph(scopes) {
       [Op.and]: [scopes.activityReport],
       calculatedStatus: REPORT_STATUSES.APPROVED,
     },
-    nest: true,
-    raw: true,
+    include: [{
+      model: ActivityReportObjective,
+      as: 'activityReportObjectives',
+      required: false,
+      include: [
+        {
+          model: Topic,
+          as: 'topics',
+        },
+      ],
+    }],
   });
 
   const topicsResponse = TOPICS.map((topic) => ({
```

Comment:
can we use the Topics table to get this instead of the constants list, as it might not be maintained as well as the DB